fe1-web/parts/connect/ConnectOpenScan.tsx

```diff
@@ -28,7 +28,9 @@ const ConnectOpenScan = ({ navigation }: IPropTypes) => {
     console.log(data);
     if (data) {
       setQrWasScanned(true);
-      navigation.navigate(STRINGS.connect_confirm_title, { laoIdIn: data });
+      const obj = JSON.parse(data);
```

Comment:
why not validate the object with `ConnectToLao.fromJson` ? 
It seems that we're actually never using the JSON Schema to validate what we scan.
And if the validation fails, a toast might be helpful :)